exporter/collector/logs.go

```diff
@@ -28,21 +28,19 @@ import (
 	"cloud.google.com/go/logging"
 	loggingv2 "cloud.google.com/go/logging/apiv2"
 	logpb "cloud.google.com/go/logging/apiv2/loggingpb"
-	"google.golang.org/genproto/googleapis/api/monitoredres"
+	"github.com/googleapis/gax-go/v2"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
```

Comment:
any specific reason to re-order these? They were grouped before by where they came from, like:
```
import(
  <standard packages>

  <google packages>

  <otel packages>

  <packages from this repo>
)
```